source/elements/oneMKL/source/domains/dft/config_params/workspace_placement.rst

```diff
@@ -0,0 +1,87 @@
+.. SPDX-FileCopyrightText: Codeplay Software
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_dft_config_workspace_placement:
+
+Workspace placement
+--------------------------------------
+
+DFT implementations often require temporary storage for intermediate data whilst computing DFTs.
+This temporary storage is referred to as a *workspace*.
+Whilst this is managed automatically by default (``config_value::WORKSPACE_AUTOMATIC`` 
+set for ``config_param::WORKSPACE_PLACEMENT``), it may be preferable to provide an external 
+workspace (``config_value::WORKSPACE_EXTERNAL`` set for ``config_param::WORKSPACE_PLACEMENT``) 
+for the following reasons:
+
+* to reduce the number of GPU mallocs / frees;
```

Comment:
I think we do not need to specify GPU here.

Reply:
Done in [84c52ba](https://github.com/oneapi-src/oneAPI-spec/pull/509/commits/84c52ba0e130300d8df2c4cb77b65008158209da).